app/models/place.rb

```diff
@@ -0,0 +1,33 @@
+class Place
```

Comment:
This is awesome - it felt really good to take 3-4 Poro's and widdle them down to 1 template.

Reply:
You guys did great